pyproject.toml

```
@@ -32,22 +32,14 @@ exclude = [
 
   [tool.poetry.dependencies]
   python = "^3.9.0"
-
-    [tool.poetry.dependencies.insegel]
-    version = "1.2.0"
-    optional = true
-
-    [tool.poetry.dependencies.sphinx]
-    version = "4.2.0"
-    optional = true
-
-    [tool.poetry.dependencies.sphinxcontrib-autoprogram]
-    version = "0.1.7"
-    optional = true
-
-    [tool.poetry.dependencies.sphinxcontrib-spelling]
-    version = "7.2.1"
-    optional = true
+  mkdocs = {extras = ["docs"], version = "^1.3.0"}
```

Docs dependencies related to the "docs" extras group, should have an `optional = true` property so they will not be installed outside the context of the "docs" group.
```suggestion
  mkdocs = {extras = ["docs"], version = "^1.3.0", optional = true}
```